redis/src/main/scala/zio/redis/api/Hashes.scala

```diff
@@ -28,9 +28,13 @@ trait Hashes {
 
   final def hLen(a: String): ZIO[RedisExecutor, RedisError, Long] = HLen.run(a)
 
-  final def hmGet(a: String, b: String, bs: String*): ZIO[RedisExecutor, RedisError, Chunk[String]] =
+  final def hmGet(a: String, b: String, bs: String*): ZIO[RedisExecutor, RedisError, Chunk[Option[String]]] =
     HmGet.run((a, (b, bs.toList)))
 
+  /** Deprecated: As per Redis 4.0.0, HMSET is considered deprecated. Please use HSET in new code. */
```

Comment:
I'd add `@param` and `@return`. It'd be good to rename these parameters as suggested in docs tickets, just to ease that work.